.gitignore

```diff
@@ -13,4 +13,5 @@
 
 # Avoid your project being littered with annoying .DS_Store files!
 .DS_Store
-.prettierignore
\ No newline at end of file
+.prettierignore
+style-guide.md
```

Comment:
add also README-template.md file

Reply:
I think also sass will be compiled to css file, this should be added also to .gitignore